src/main/java/com/puppycrawl/tools/checkstyle/utils/CommonUtils.java

```diff
@@ -101,6 +113,112 @@ public static Pattern createPattern(String pattern, int flags) {
     }
 
     /**
+     * Creates DetailAST block comment to pass it to the Javadoc parser.
```

Comment:
Method is in `CommonUtils` now, so it is meant to be used anywhere, not only in Javadoc parser. Please update javadoc for this method.

Reply:
done